motoko/defi/dfx.json

```diff
@@ -32,20 +32,15 @@
       "main": "src/DIP20/motoko/src/token.mo"
     }
   },
+  "networks": {
+    "local": {
+      "bind": "127.0.0.1:8000"
+    }
+  },
```

Comment:
Does this test work without a defined local network?

Reply:
No, because it requires subnet type `system`, and I don't want to mess with networks.json on behalf of the user if they run the install script